File: 11 Która funkcja poprawnie zwraca numer litery w alfabecie angielskim (26 liter) (Wielkie ASCII 65 - 90, Małe 97 - 122)/treść.py
def A(litera):
    litera_ascii = litera[0]
    litera_ascii = ord(litera_ascii)
    if litera_ascii >= 65 and litera_ascii <= 90:
        return litera_ascii - 64
    else:
        litera_ascii = litera_ascii - 25
        litera_ascii = litera_ascii - 7
    return litera_ascii - 64

def B(litera):
    if litera.islower():
        return chr(litera) - 26 - 65
    else:
        return chr(litera) - 65

def C(litera):
    return litera if litera.isupper() else litera - 26

def D(litera:str):
    return ord(litera.upper()) - 64

File: 12 Która funkcja poprawnie zwraca numer litery w alfabecie angielskim (26 liter) (Wielkie ASCII 65 - 90, Małe 97 - 122)/treść.cpp
int A(char litera)
{
    int litera_ascii;
    litera_ascii = (int)litera;
    if(litera_ascii >= 65 && litera_ascii <= 90)
    {
        return litera_ascii - 64;
    }
    else
    {
        litera_ascii = litera_ascii - 25;
        litera_ascii = litera_ascii - 7;
    }
    return litera_ascii - 64;
}

char B(char litera)
{
    if(islower(litera))
        return litera - 26 - 65;
    else
        return litera - 65;
}

void C(char litera)
{
    return isupper(litera) ? litera : litera - 26;
}

int D(char litera)
{
    return toupper(litera) - 64;
}